src/Microsoft.ML.Core/Data/KeyType.cs

```diff
@@ -73,16 +80,31 @@ public override bool Equals(DataViewType other)
             return true;
         }
 
+        /// <summary>
+        /// Determine if a KeyType instance is equal to another KeyType instance.
+        /// Checks if any object is the type of KeyType, if the RawType
+        /// is the same, and if the Count is the same.
+        /// </summary>
+        /// <param name="other">The other object to compare against.</param>
+        /// <returns>A boolean if both objects are equal or not.</returns>
```

Comment:
> /// <returns>A boolean if both objects are equal or not.</returns [](start = 7, length = 66)

for bool returns, please document them like this: [https://github.com/dotnet/machinelearning/blob/6e9023fa08ee2dba2d9b43e5485b7230c6dd29ed/src/Microsoft.ML.StandardLearners/Standard/Online/AveragedLinear.cs#L94](https://github.com/dotnet/machinelearning/blob/6e9023fa08ee2dba2d9b43e5485b7230c6dd29ed/src/Microsoft.ML.StandardLearners/Standard/Online/AveragedLinear.cs#L94)